routers/api/v1/repo/issue_reaction.go

```diff
@@ -114,8 +114,10 @@ func PostIssueCommentReaction(ctx *context.APIContext, form api.EditReactionOpti
 	//   schema:
 	//     "$ref": "#/definitions/EditReactionOption"
 	// responses:
+	//   "200":
+	//     "$ref": "#/responses/Reaction"
 	//   "201":
-	//     "$ref": "#/responses/ReactionResponse"
+	//     "$ref": "#/responses/Reaction"
```

Comment:
Return both 200 and 201?

Reply:
no this is jus the swagger definition wich tell us that this API **is expect** to return this *vaules*
so it tells the programmer make sure you handle StatusOK and StatusCreated

Reply:
@bagasme 
no this is jus the swagger definition wich tell us that this API **is expect** to return this *vaules*
so it tells the programmer make sure you handle StatusOK and StatusCreated